### PR TITLE
Update Experimentors and make final assault more challenging

### PR DESCRIPTION
**Pull request by MatGB**
Based on Sloth's work, added Experiment Zero monster and changed defensive numbers of outpost world

### CODE REVIEW

File: default/buildings.txt [hunk 1]
```diff
@@ -625,7 +625,7 @@ BuildingType
             stackinggroup = "INDUSTRY_CENTER_STACK"
             effects = SetTargetIndustry value = Value + Target.Population * 3 * [[INDUSTRY_PER_POP]]
     ]
-    icon = ""
+    icon = "icons/tech/industrial_centre_ii.png"
```
Comment (Cjkjvfnby) [inline default/buildings.txt:628]: I want to mention about workflow:
This changes was done in separate commit and it is good. But this commit is part of this pull request and I don't like it. I suggest to make separate pr for this changes.

Comment (MatGB) [inline default/buildings.txt:628]: Don't understand a) what exactly you mean there and more importantly b) how to do it, I've tried rebasing and squashing stuff, in this particular case I was testing Sloth's patch while working on the buildings and didn't realise Git would put them together the way it has, I know now but it's still the way it'd work out normally.

Comment (Cjkjvfnby) [inline default/buildings.txt:628]: I mean that I want to have 2 pull requests for this two features.  
I mean that then you see diff for pull request and see that you have two features you will not create it.
Please forget **rebasing** and **squashing** words for a while.  You don't feel comfort on basic git operation. 

Process of adding feature is next:
- update you master to actual state
- create new branch (you did not do it then apply changes form Sloth thats why you got mix)
- make your feature (or merge changes from other branch) and dont forget to merge actual master in during development
- Push (all rebasing and squashing can be done before it step)
- Make pr (Always check diff before creating pr!)

Comment (Vezzra) [inline default/buildings.txt:628]: @MatGB: What Cj said. Basically don't do any dev work in master, but create a branch for each feature your working on (except for very small, quick fixes which can be done in a single commit, in that case the create branch - implement fix - merge into master workflow is a bit of an overkill).

Comment (MatGB) [inline default/buildings.txt:628]: Ah, OK, never commit from a master branch, that makes some sense, I'll have to figure out the taskflow but I know what you're asking for now. I guess it's because I do most of my work on the fly while playing, just adding a quick fix to a problem, especially the icons or stringtable updates, I'll get there, thanks for the explanation.
